.eslintrc.js

```diff
@@ -1,31 +1,39 @@
 module.exports = {
-    parserOptions: {
-      ecmaVersion: 2019,
-      sourceType: "module",
-    },
-    env: {
-      browser: true,
-      es6: true,
-    },
-    extends: ["airbnb-base"],
-    plugins: ["jest", "svelte3"],
-    rules: {
-      "import/prefer-default-export": "off",
-      "import/no-extraneous-dependencies": ["error", { devDependencies: true }],
-      "no-param-reassign": [
-        "error",
-        { props: true, ignorePropertyModificationsFor: ["draft"] },
-      ],
+  parserOptions: {
+    ecmaVersion: 2019,
+    sourceType: "module",
+  },
+  env: {
+    browser: true,
+    es6: true,
+  },
+  extends: ["airbnb-base"],
+  plugins: ["jest", "svelte3"],
+  rules: {
```

Comment:
does this need the same check as https://github.com/graph-paper-org/graph-paper/pull/40/files#diff-e4403a877d80de653400d88d85e4801aR37?

Reply:
Yes, although that's not related to Prettier. I'll open another PR for getting the template to pass ESLint tests.